website/src/pages/api/messages/user.ts

```diff
@@ -4,6 +4,7 @@ const handler = withoutRole("banned", async (req, res, token) => {
   //TODO: add params if needed
   const params = new URLSearchParams({
     username: token.sub,
+    auth_method: "local",
```

Comment:
Can we make use of `getBackendUserCore` (from `src/lib/users`) and use whatever `auth_method` that returns?  username and auth method are a little depending on if the user signed in with discord or not.